### ISSUE TITLE
`x` doesn't work in search bars

### ISSUE THREAD

**Issue by srnyx** [killpowa/Twickt-Launcher#1366]
**Describe the bug**
The `x` on the right side of search bars doesn't work. This is in all search bars, not just the one shown below.

**To Reproduce**
1. Type something in a search bar
2. Click the `x`
3. Observe bug

**Expected behavior**
The text that is typed should be removed.

**Screenshots**
![](https://im2.ezgif.com/tmp/ezgif-2-9482301a50.gif)

**Operating System:**
- OS: Windows 10
- Version: 1.1.24
- Java Version: GraalVM 17

**Comment by AshtakaOOf** [comment]
Yeah that is dumb...
Also
> Version: 10

This is the gdlauncher version not the OS version

**Comment by srnyx** [comment]
> > Version: 10
> 
> This is the gdlauncher version not the OS version

Oh... 😳

**Comment by AshtakaOOf** [comment]
Oh wait am i dumb?

**Comment by srnyx** [comment]
> Oh wait am i dumb?

In the issue template it gives `10` as an example, so I think it is referring to the OS version, but I'll keep it like this cause I assume they still need the GDLauncher version lol

**Comment by AshtakaOOf** [comment]
It must be the gdlauncher version, if not how could you debug stuff like that?
It's weird that it is under the "Operating System:" but anyway.

**Comment by Eskaan** [comment]
So, which GDL version do you have? (Settings/Gneral at the bottom) and could you show us your console after clicking `x`?

**Comment by srnyx** [comment]
> So, which GDL version do you have?

1.1.24, I put it under **Operating System**

> and could you show us your console after clicking `x`?

```
Uncaught TypeError: Cannot set property target of #<Event> which has only a getter
    at O (Input.js:65:11)
    at r.handleReset [as clickfalse] (Input.js:168:7)
    at HTMLSpanElement.P (props.js:153:20)
O @ Input.js:65
r.handleReset @ Input.js:168
P @ props.js:153
```

**Comment by LordDeatHunter** [comment]
Is there a release with the fix? I'm still getting it in v1.1.25
![image](https://user-images.githubusercontent.com/26803198/179392606-49772d10-e695-41fc-a7d9-a64d872c848c.png)

**Comment by Eskaan** [comment]
Next release, the fix is merged.
